src/index.js

```diff
@@ -0,0 +1,16 @@
+import createProject from './project-model';
+import createTodo from './todo-model';
+import app from './app-model';
+import insertAllProjects from './dom-manipulation';
+import './style.css';
+import './modal.css';
+
+// let CreateTodo = require('./todo-model')
+
```

Comment:
- [ ] Kindly remove all the commented code in your project. This will keep your project clean and professional. Remember you can always use git history to go back to the previous version of your code ⬇️ 
<img width="792" alt="no_inline_comments" src="https://user-images.githubusercontent.com/32361963/117766481-52ae4f00-b238-11eb-8936-08b93c9dcb67.png">